stdpopsim/ext/selection.py

```diff
@@ -1,107 +1,6 @@
 import attr
 
 
-@attr.s(kw_only=True)
```

Comment:
Not much left in this file.... can we ditch it?

Reply:
I dunno - there's a fair bit still there, I think! And I don't want to deal with rearranging that stuff in this PR.

Reply:
yeah, we should move it somewhere else once we decide on an API for sweeps, I think.